sunpy/coordinates/tests/test_ephemeris.py

```diff
@@ -111,3 +112,17 @@ def test_get_sun_orientation():
     # Check the Southern Hemisphere
     angle = get_sun_orientation(EarthLocation(lat=-40*u.deg, lon=-75*u.deg), '2017-02-18 13:00')
     assert_quantity_allclose(angle, -110.8*u.deg, atol=0.1*u.deg)
+
+
+@pytest.mark.remote_data
+def test_get_horizons_coord():
```

Comment:
We need a way of marking this test as needs astroquery. Then we should add astroquery to our online test deps.

Reply:
pytest importer skip?

Reply:
Okay, and where do I add astroquery as a dependency for online tests?

Reply:
In tox.ini?

Reply:
Yeah that would work. 

Adding it `online: astropyquery` to the last part of the `deps=` list should work.